lib/ddtrace/contrib/redis/patcher.rb

```diff
@@ -72,16 +73,28 @@ def call_pipeline(*args, &block)
             def datadog_pin
               @datadog_pin ||= begin
                 pin = Datadog::Pin.new(
-                  Datadog.configuration[:redis][:service_name],
+                  datadog_configuration[:service_name],
                   app: Ext::APP,
                   app_type: Datadog::Ext::AppTypes::DB,
                   tracer: Datadog.configuration[:redis][:tracer]
```

Comment:
I may need to use ici as well the custom configuration tracer (since it could be different)